src/main/scala/is/hail/variant/Genotype.scala

```diff
@@ -309,11 +309,18 @@ object Genotype {
       gtPairSqrt(i)
   }
 
+  def gtPairPhased(i: Int): GTPair = {
```

Comment:
Just a meta comment: I like adjectives before the nouns they modify, just like English: big red dog, phased allele pair.